.changeset/smooth-ducks-sell.md

```diff
@@ -0,0 +1,9 @@
+---
+"@latticexyz/common": patch
```

Comment:
this should prob be minor but we're trying to batch a few minor version changes and don't want to block patch changes